### PR TITLE
[insights][docs] add brief insights alert docs

### PR DESCRIPTION
**Pull request by benpankow**
## Summary
Adds brief Insights alerting docs which lay out what can be alerted on and provides some examples of use-cases.

Screenshot 2024-04-04 at 3 12 42 PM

## Test Plan

Local docs.

### CODE REVIEW

**Comment by benpankow** [comment]
* **#21038** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21038?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @benpankow and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/content/dagster-cloud/managing-deployments/alerts.mdx [hunk 1]
```diff
@@ -102,6 +102,31 @@ Alert policies are configured on a **per-deployment basis**. This means, for exa
         <a href="/dagster-cloud/deployment/hybrid">Hybrid deployments</a> only.
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Insights metric usage alert</strong>
+      </td>
+      <td>
+        <Note>A Dagster Cloud Pro plan is required to use this feature.</Note>
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/managing-deployments/alerts.mdx:110]: I think it's fine to put this as the first sentence in the paragraph and bold it over using a callout. Looks a lil odd in the table.

Ex: `**This feature requires a Dagster Cloud Pro plan.**`

**Review by erinkcochran87 — APPROVED** [review]
Just one comment about formatting, but otherwise lgtm! Thank you 🙏

**Review by erinkcochran87 — APPROVED** [review]
We should add **Experimental** in here somewhere to match the UI, but otherwise lgtm

**Pull request merged**